WooCommerce/src/androidTest/kotlin/com/woocommerce/android/screenshots/orders/SingleOrderScreen.kt

```diff
@@ -18,20 +25,24 @@ class SingleOrderScreen : Screen {
         return OrderListScreen()
     }
 
-    fun assertSingleOrderScreen(): SingleOrderScreen {
+    private fun assertIdAndTextDisplayed(id: Int, text: String?) {
+        Espresso.onView(
+            Matchers.allOf(
+                ViewMatchers.withId(id), ViewMatchers.withText(text)
+            )
+        ).check(ViewAssertions.matches(ViewMatchers.isDisplayed()))
+    }
+
+    fun assertSingleOrderScreenWithProduct(order: OrderData): SingleOrderScreen {
         Espresso.onView(withId(R.id.toolbar))
-            .check(ViewAssertions.matches(hasDescendant(withSubstring("#"))))
-            .check(ViewAssertions.matches(isDisplayed()))
-        Espresso.onView(withId(ORDER_NUMBER_LABEL))
-            .check(ViewAssertions.matches(isDisplayed()))
-        Espresso.onView(withId(R.id.paymentInfo_total))
+            .check(ViewAssertions.matches(hasDescendant(withSubstring("#" + order.id))))
             .check(ViewAssertions.matches(isDisplayed()))
-        return this
-    }
 
-    fun assertSingleOrderScreenWithProduct(productName: String): SingleOrderScreen {
-        Espresso.onView(withText(productName)).check(ViewAssertions.matches(isDisplayed()))
-        return assertSingleOrderScreen()
+        Espresso.onView(withText(order.productName)).check(ViewAssertions.matches(isDisplayed()))
```

Comment:
Nitpick: just to conform the indentation in the rest of the file:

```suggestion
        Espresso.onView(withText(order.productName))
            .check(ViewAssertions.matches(isDisplayed()))
```

Reply:
Updated in https://github.com/woocommerce/woocommerce-android/pull/7302/commits/841bbae63d88cb05ddc3ee8d00c5cb31d7e63b7b